pkgs/apps/xtb/default.nix

```diff
@@ -50,10 +50,11 @@ in stdenv.mkDerivation rec {
 
   postFixup = ''
     wrapProgram $out/bin/xtb \
-      --prefix PATH : "${binSearchPath}" \
-      --set-default XTBPATH "$out/share/xtb"
+      --prefix PATH : "${binSearchPath}"
   '';
 
+  setupHooks = [ ./xtbHook.sh ];
```

Comment:
This is for python + nix-shell?

Reply:
Yes and no. It also is important for Pysisyphus GFN-0 (and other cases like this) calculations by XTB-Python. The C-API and Python API will also not work for GFN-0, if this variable is not set, even with a regular `nix-build`. The old version with the wrapper just around the executable does not work, as the executable itself will never be called.